assets/_scss/default/body-main-styles.scss

```diff
@@ -26,13 +26,14 @@ html {
 body {
   color: var(--body-text-color);
```

Comment:
where are these vars defined, can't see the file in this PR

Reply:
on the /assets/_scss/colors/dark-colors.scss